packages/replay-internal/src/replay.ts

```diff
@@ -1226,27 +1226,29 @@ export class ReplayContainer implements ReplayContainerInterface {
       // TODO FN: Evaluate if we want to stop here, or remove this again?
     }
 
-    // this._flushLock acts as a lock so that future calls to `_flush()`
-    // will be blocked until this promise resolves
+    const _flushInProgress = !!this._flushLock;
```

Comment:
what does the !! do?

Reply:
Ah it's a shortcut to coerce to a boolean instead of e.g. `Boolean()`